### PR TITLE
Adapt heading relied on the navigation item 

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
Fixes https://github.com/nextcloud/activity/issues/1464

|  Before   | After                                                        |
|-------------|---------------------------------------------------------------|
| Screenshot from 2024-01-15 16-28-17 | Screenshot from 2024-01-15 16-23-08

### CODE REVIEW

File: src/views/ActivityAppFeed.vue [hunk 1]
```diff
@@ -21,7 +21,7 @@
 <template>
 	<NcAppContent class="activity-app">
 		<h1 class="activity-app__heading">
-			{{ t('activity', 'Your activities') }}
+			{{ t('activity', `${headingTitle}`) }}
```
Comment (ShGKme) [inline src/views/ActivityAppFeed.vue:24]: It is already translated

```suggestion
			{{ headingTitle }}
```

**Review by ShGKme — APPROVED** [review]

**Review by artonge — APPROVED** [review]

**Review by susnux — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Pull request merged**